internal/exec/engine.go

```diff
@@ -67,6 +71,26 @@ func (e Engine) Run(stageName string) error {
 		Ignition: types.Ignition{Version: types.MaxVersion.String()},
 	}
 
+	if stageName == "fetch" && e.DetectOfflineConfig != "" {
```

Comment:
We should fix the Ignition code so that the stages can have their own CLI args.

Reply:
:+1: I can't agree more.